osu.Game/Localisation/NamedOverlayComponentStrings.cs

```diff
@@ -0,0 +1,66 @@
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+using osu.Game.Resources.Localisation.Web;
+
+namespace osu.Game.Localisation
+{
+    public static class NamedOverlayComponentStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.NamedOverlayComponent";
+
+        /// <inheritdoc cref="PageTitleStrings.MainBeatmapsetsControllerIndex"/>
+        public static LocalisableString BeatmapListingTitle => PageTitleStrings.MainBeatmapsetsControllerIndex;
+
+        /// <summary>
+        /// "browse for new beatmaps"
+        /// </summary>
+        public static LocalisableString BeatmapListingDescription => new TranslatableString(getKey(@"beatmap_listing_description"), @"browse for new beatmaps");
+
+        /// <inheritdoc cref="PageTitleStrings.MainBeatmapsetsControllerShow"/>
+        public static LocalisableString BeatmapSetTitle => PageTitleStrings.MainBeatmapsetsControllerShow;
+
+        /// <inheritdoc cref="PageTitleStrings.MainChangelogControllerDefault"/>
+        public static LocalisableString ChangelogTitle => PageTitleStrings.MainChangelogControllerDefault;
+
+        /// <summary>
+        /// "track recent dev updates in the osu! ecosystem"
+        /// </summary>
+        public static LocalisableString ChangelogDescription => new TranslatableString(getKey(@"changelog_description"), @"track recent dev updates in the osu! ecosystem");
+
+        /// <inheritdoc cref="PageTitleStrings.MainHomeControllerIndex"/>
+        public static LocalisableString DashboardTitle => PageTitleStrings.MainHomeControllerIndex;
+
+        /// <summary>
+        /// "view your friends and other information"
+        /// </summary>
+        public static LocalisableString DashboardDescription => new TranslatableString(getKey(@"dashboard_description"), @"view your friends and other information");
+
+        /// <inheritdoc cref="PageTitleStrings.MainRankingControllerDefault"/>
+        public static LocalisableString RankingsTitle => PageTitleStrings.MainRankingControllerDefault;
+
+        /// <summary>
+        /// "find out who's the best right now"
+        /// </summary>
+        public static LocalisableString RankingsDescription => new TranslatableString(getKey(@"rankings_description"), @"find out who's the best right now");
+
+        /// <inheritdoc cref="PageTitleStrings.MainNewsControllerDefault"/>
+        public static LocalisableString NewsTitle => PageTitleStrings.MainNewsControllerDefault;
+
+        /// <summary>
+        /// "get up-to-date on community happenings"
+        /// </summary>
+        public static LocalisableString NewsDescription => new TranslatableString(getKey(@"news_description"), @"get up-to-date on community happenings");
+
+        /// <inheritdoc cref="PageTitleStrings.MainWikiControllerDefault"/>
+        public static LocalisableString WikiTitle => PageTitleStrings.MainWikiControllerDefault;
```

Comment:
Why are these here? Just use the osu-web ones directly, I think?

Reply:
`PageTitleStrings` comes from `osu.Game.Resources.Localisation.Web` namespace.

---

and as discussed before, the `*Title` and `*Description` style is for consistency:
```cs
//in components
{
    Title = NamedOverlayComponentStrings./* name here */Title;
    Description = NamedOverlayComponentStrings./* name here */Description;
}
//in osu.Game/Localisation/NamedOverlayComponentStrings.cs
    /// <inheritdoc cref="PageTitleStrings.{string name}"/>
    public static LocalisableString /* name here */Title => PageTitleStrings./* name here */ControllerDefault;

    /// <summary>
    /// "sample string"
    /// </summary>
    public static LocalisableString /* name here */Description => new TranslatableString(getKey(@"_description"), @"sample string");
```

But it's ok using following style :shrug:
```cs
//in components
{
    Title = PageTitleStrings./* name here */ControllerDefault;
    Description = NamedOverlayComponentStrings./* name here */Description;
}
```

Reply:
I definitely prefer the direct references. @smoogipoo can you confirm your opinion?

Reply:
I also prefer direct references.

Reply:
👌🏼 it will be direct references in next commit

Reply:
Not sure if it will ever be the case, but I can see these potentially being moved to `osu-web` in the future (if there are more design updates to bring web/client in closer consistency).

Reply:
Yeah consistency is really the only reason I've went with that style, but since redirecting in generated strings classes is not a good idea in general, I'm fine with having it use direct references.

Moving the tooltip descriptions web-side would also remove all this hassle and leave things in a consistent state, but that's probably for later when actually needed.